script/setup.js

```diff
@@ -181,7 +181,7 @@ try {
 	console.log();
 	console.log(chalk.gray`Generating all-contributors table in README.md...`);
 
-	await $`npx all-contributors generate`;
+	await $`npx --yes all-contributors generate`;
```

Comment:
Aha! So this was the culprit! 

Looking at the other used in this script, they're all added as `devDependency` entries in `package.json`. `all-contributors` is the only one installed with `npx`. What do you think about installing it as a dev dependency @jsjoeio? I'm leaning towards that so the setup script doesn't have to install any additional packages.

If so, note that it'll need to be uninstalled in the `pnpm uninstall` command later on in the file, as it wouldn't be used by users of the package.

If not, this is good to merge IMO!

Reply:
That sounds good to me!